templates/js/sso-tab-with-obo-flow/infra/azure.bicep

```diff
@@ -146,19 +123,9 @@ resource authSettings 'Microsoft.Web/sites/config@2021-02-01' = {
   }
 }
 
-// Azure Storage is required when creating Azure Functions instance
-resource functionStorage 'Microsoft.Storage/storageAccounts@2021-06-01' = {
```

Comment:
Why does the function app not need a storage?

Reply:
Sounds like Azure Funcion will create a App Service Plan provide 10GB storage here.